StoreSpidLogs/handler.ts

```diff
@@ -4,14 +4,18 @@ import {
   toEncryptedPayload
 } from "@pagopa/ts-commons/lib/encrypt";
 import { IPString, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
-import { curry } from "fp-ts/lib/function";
+
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+
 import * as t from "io-ts";
 import { SpidMsgItem } from "./index";
 
 import { UTCISODateFromString } from "@pagopa/ts-commons/lib/dates";
 import { readableReport } from "@pagopa/ts-commons/lib/reporters";
 import { sequenceS } from "fp-ts/lib/Apply";
-import { either } from "fp-ts/lib/Either";
+
+import { curry2 } from "fp-ts-std/Function";
```

Comment:
Do we need this one?

Reply:
It's used a little below. I was asking myself the same.

Reply:
Yes, I replaced a curring function that doesn't anymore exists. 
But we can for sure remove it and implement in another way if you thinks it's better to avoid a new library

Reply:
@balanza no, obviously it's not a dev dependency. I'll remove it

Reply:
Removed fp-ts-std [here](https://github.com/pagopa/io-functions-app/pull/202/commits/11eaa89932ef7961eee0fbf344499f46a212b509)